pkgs/development/libraries/beignet/default.nix

```diff
@@ -0,0 +1,117 @@
+{ pkgs
```

Comment:
Best practice is to pass in what you need explicitly, not the entire package set.